gtsfm/two_view_estimator.py

```diff
@@ -293,7 +293,7 @@ def run_2view(
         )
 
         # Optionally, do two-view bundle adjustment
-        if self._bundle_adjust_2view:
+        if self._bundle_adjust_2view and len(pre_ba_v_corr_idxs) >= self.processor._min_num_inliers_est_model:
```

Comment:
I'd prefer this to a be logged (at least to debug) rather than a silent skip: 
```
if len(pre_ba_v_corr_idxs) < self.processor._min_num_inliers_est_model: 
  logger.info("Insufficient verified correspondences for 2-view BA, skipping")
```

Reply:
Why do we need to log this? Either way it will just get thrown out by the InlierSupportProcessor. This just avoids the extra BA step on something that would otherwise be thrown away.